pkg/server/message.go

```diff
@@ -283,24 +283,24 @@ func (t *TranslateServiceServer) fuzzyTranslate(
 	}
 
 	for i := range all {
-		// Skip original messages
+		// Skip original translation
 		if i == origIdx {
 			continue
 		}
 
-		// Create a map to store pointers to untranslated messages
+		// Create a map to store pointers to untranslated translation
 		untranslatedMessagesLookup := make(map[string]*model.Message)
 
-		// Iterate over the messages and add any untranslated messages to the untranslated messages lookup
+		// Iterate over the translation and add any untranslated translation to the untranslated translation lookup
```

Comment:
// Iterate over the messages and add any untranslated messages to the untranslated messages lookup

Reply:
+1